task-launcher/src/tasks/hearts-and-flowers/trials/stimulus.js

```diff
@@ -22,26 +28,11 @@ export function stimulus(isPractice = false, stage, stimulusDuration, postTrialG
       };
     },
     stimulus: () => {
-      return `<div id='stimulus-container-hf'>
-                    <div class='stimulus'>
-                        ${
-                          jsPsych.timelineVariable('position') <= 0.5
-                            ? `<img src=${
-                                mediaAssets.images[jsPsych.timelineVariable('stimulus')]
-                              } alt="heart or flower"/>`
-                            : ''
-                        }
-                    </div>
-                    <div class='stimulus'>
-                        ${
-                          jsPsych.timelineVariable('position') > 0.5
-                            ? `<img src=${
-                                mediaAssets.images[jsPsych.timelineVariable('stimulus')]
-                              } alt="heart or flower"/>`
-                            : ''
-                        }
-                    </div>
-                </div>`;
+      return getLayoutTemplate(
+        null,
+        mediaAssets.images[jsPsych.timelineVariable('stimulus')],
+        jsPsych.timelineVariable('position') <= 0.5
+      );
     },
     stimulus_duration: stimulusDuration,
     post_trial_gap: isPractice? null : postTrialGap,
```

Comment:
@anabonne the `stimulus_duration` is causing the container to disappear after sometime. Is that the expected behavior and the `post_trial_gap` is causing a flicker on the screen as well. @kachergis @Zio-4  thoughts?

Reply:
as discussed, good to not have the container disappear -- thank you!

Reply:
Yep we will need to make only the heart or flower disappear. For now we can just remove any timeouts on the stimulus.